cmd/thanos/store.go

```diff
@@ -101,6 +115,8 @@ func runStore(
 	verbose bool,
 	syncInterval time.Duration,
 	blockSyncConcurrency int,
+	minTime *store.TimeOrDurationValue,
```

Comment:
Hmmm I let's maybe put this `TimeOrDurationValue` to separate package? Can be useul for other components like compactor, querier etc

Reply:
Plust @brancz mentioned use cases applies

Reply:
Moved to model package